openassessment/xblock/static/js/src/lms/oa_peer.js

```diff
@@ -159,13 +154,7 @@ OpenAssessment.PeerView.prototype = {
      >> true
      **/
     peerSubmitEnabled: function(enabled) {
-        var button = $('.peer-assessment--001__assessment__submit', this.element);
```

Comment:
Yes, it was satisfying (though also frightening that previous authors felt OK about duplicating the same thing 9 times).